core/market/decentralized/src/market.rs

```diff
@@ -103,8 +104,12 @@ impl MarketService {
             .extend(rest_api::requestor::register_endpoints)
     }
 
-    pub async fn get_offers(&self, id: Option<Identity>) -> Result<Vec<Offer>, MarketError> {
-        Ok(self.matcher.store.get_offers(id).await?)
+    pub async fn get_offers(
+        &self,
+        id: Option<Identity>,
+        validation_ts: NaiveDateTime,
```

Comment:
`validation_ts` is not needed at Market level. Should be enclosed within `SubscriptionStore`